OpenRA.Mods.Common/Traits/Render/WithDamageOverlay.cs

```diff
@@ -44,6 +49,24 @@ public class WithDamageOverlayInfo : TraitInfo, Requires<RenderSpritesInfo>
 		public readonly DamageState MinimumDamageState = DamageState.Heavy;
 		public readonly DamageState MaximumDamageState = DamageState.Dead;
 
+		public IEnumerable<string> GetSequences(MersenneTwister random)
+		{
+			if (!string.IsNullOrEmpty(StartSequence))
+				yield return StartSequence;
+
+			if (!string.IsNullOrEmpty(LoopSequence))
+			{
+				var loopCount = LoopCount.Length == 2 ? random.Next(LoopCount[0], LoopCount[1] + 1) : LoopCount[0];
```

Comment:
Perhaps validate if not larger than 2.

Reply:
Random will raise an exception when low is larger than high. Perhaps use min and max.

Seems odd to me that the end of range is inclusive. Perhaps mention it in the comments.

Reply:
I think it's only worth a comment if it's an exclusive range.